--- origsrc/qwt-5.1.1/qwtconfig.pri	2008-07-15 22:54:16.906250000 -0500
+++ src/qwt-5.1.1/qwtconfig.pri	2008-07-15 22:56:43.953125000 -0500
@@ -15,9 +15,9 @@
     INSTALLBASE    = C:/Qwt-5.1.1
 }
 
-target.path    = $$INSTALLBASE/lib
-headers.path   = $$INSTALLBASE/include
-doc.path       = $$INSTALLBASE/doc
+target.path    = $$[QT_INSTALL_LIBS]
+headers.path   = $$[QT_INSTALL_HEADERS]
+doc.path       = /usr/share/doc/qwt5
 
 ######################################################################
 # qmake internal options
