--- origsrc/qwt-5.2.2/qwtconfig.pri	2011-08-01 04:28:22.000000000 -0500
+++ src/qwt-5.2.2/qwtconfig.pri	2011-08-04 19:58:13.859424500 -0500
@@ -15,9 +15,9 @@ win32 {
     INSTALLBASE    = C:/Qwt-$$VERSION
 }
 
-target.path    = $$INSTALLBASE/lib
-headers.path   = $$INSTALLBASE/include
-doc.path       = $$INSTALLBASE/doc
+target.path    = $$[QT_INSTALL_LIBS]
+headers.path   = $$[QT_INSTALL_HEADERS]
+doc.path       = /usr/share/doc/qwt$${VER_MAJ}
 
 ######################################################################
 # qmake internal options
@@ -98,7 +98,7 @@ CONFIG     += QwtWidgets
 # QwtSVGItem.
 ######################################################################
 
-#CONFIG     += QwtSVGItem
+CONFIG     += QwtSVGItem
 
 ######################################################################
 # You can use the MathML renderer of the Qt solutions package to 
