--- origsrc/qwt-5.2.2/textengines/mathml/mathml.pro	2011-08-01 04:28:22.000000000 -0500
+++ src/qwt-5.2.2/textengines/mathml/mathml.pro	2011-08-04 19:55:27.618916000 -0500
@@ -19,10 +19,6 @@ isEmpty(VVERSION) {
 
     # Qt4
 
-    include( ../textengines.pri )
-
-    exists( qtmmlwidget.cpp ) {
-
         TARGET    = qwtmathml$${SUFFIX_STR}
         VERSION   = 1.0.0
         QT       += xml
@@ -33,6 +29,7 @@ isEmpty(VVERSION) {
         SOURCES = \
             qwt_mathml_text_engine.cpp
 
+    exists( qtmmlwidget.cpp ) {
         # The files below can be found in the MathML tarball of the Qt Solution 
         # package http://www.trolltech.com/products/qt/addon/solutions/catalog/4/Widgets/qtmmlwidget
         # that is available for owners of a commercial Qt license.
@@ -44,6 +41,9 @@ isEmpty(VVERSION) {
     }
 
     else {
-        error( "qtmmlwidget.cpp is missing, see http://www.trolltech.com/products/qt/addon/solutions/catalog/4/Widgets/qtmmlwidget" )
+        INCLUDEPATH += $$[QT_INSTALL_HEADERS]/QtMmlWidget
+        LIBS += -lQtSolutions_MMLWidget-2.4
     }
+
+    include( ../textengines.pri )
 }
